app/avo/resources/concerns/avo_auditable_resource.rb

```diff
@@ -1,22 +1,21 @@
 module Avo::Resources::Concerns::AvoAuditableResource
   extend ActiveSupport::Concern
 
-  class_methods do
-    def inherited(subclass)
-      super
+  def fetch_fields
+    super
+    return unless view.form?
 
-      subclass.concerning :AuditableFields, prepend: true do
-        def fields # rubocop:disable Lint/NestedMethodDefinition
-          raise "fields called"
-          super
+    field :comment, as: :textarea, required: true,
+          help: "A comment explaining why this action was taken.<br>Will be saved in the audit log.<br>Must be more than 10 characters."
+  end
+
+  # Would be nice if there was a way to force a field to show up as visible
```

Comment:
cc @adrianthedev this seems to be something where avo3 is stricter than avo2, and it would be nice if there was a way to force a field to be visible even if avoiding thinks it shouldn't be

Reply:
Hey @segiddins. Can you please share what is the problem? Maybe I can figure out an easier fix for your use case and remove this monkeypatch.

Reply:
we have fields that we need to show up in forms, even though Avo considers them "computed" (because the underlying record has no setter for them)